README.md

```diff
@@ -1 +1,3 @@
-# hello-microverse
+# Hello Microverse
+
+My first project at Microverse. The main goal of this project is to learn the basics of Github workflow, linters and Code Review flow.
```

Comment:
- [ ] It seems you got a bit confused here, I'm gonna leave a screenshot of the requirement below, so kindly use the template from this [repo](https://github.com/microverseinc/readme-template),, to copy it, click on the README.md file in the repo I linked to you, click in RAW here:
![image](https://user-images.githubusercontent.com/73140461/143136919-fccbaef2-c6c0-40d0-a997-d20038fe4901.png)
and then copy the entire template to your README file and fill it with relevant information for this project.

![image](https://user-images.githubusercontent.com/73140461/143136758-b8b669c1-8fd8-47f9-966f-5667d9b55aec.png)

Reply:
I got confused with the requirements. I did not scroll down and thought that only the Header and paragraph are necessary for the Readme file.

Reply:
@leolpaz thank you for the explanation with the picture you send me.

Reply:
@leolpaz please send me additional information on how to finish the Code Review process successfully.